docs/docs/reference/singleton-types.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: doc-page
+title: "Literal Singleton Types"
+---
+
+Literal Singleton Types allows primitive literals to be used as types. For example:
+
+```scala
+val t: "Jedi".type = "Jedi"
+val t: 42.type = 42
```

Comment:
Um, that's why I had a separate section for Dotty earlier. One minute, will revert changes and push again.